flow/size.yaml

```diff
@@ -26,6 +30,23 @@ components:
         random:
           $ref: '#/components/schemas/Flow.SizeRandom'
           x-field-uid: 4
+        weight_pairs:
+          $ref: '#/components/schemas/Flow.SizeWeightPairs'
+          x-field-uid: 5
+        quad_gaussian:
```

Comment:
do we need to be fix the size to 4 and call it quad ? shouldn't it just be generic with minItems=1 ?

I didn't quite get what we mean by "normalized to single distribution". To me it looks more like size-weight pairs, just that `fixed size` frame size has been replaced with range of frame sizes represented by `gaussian curve`.

Reply:
removed the fixed limit.